package.json

```
@@ -5,28 +5,34 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "tsc && vite build",
     "preview": "vite preview",
     "lint": "eslint src/**/*.{js,jsx,ts,tsx}",
-    "lint:fix": "yarn run lint --fix",
+    "lint:fix": "pnpm run lint --fix",
     "prettier": "prettier --check src/**/*.{js,jsx,ts,tsx,css,scss,json}",
-    "prettier:fix": "yarn run prettier --write",
-    "prepare": "husky install"
+    "prettier:fix": "pnpm run prettier --write",
+    "prepare": "husky install",
+    "typeCheck": "tsc"
```

```suggestion
    "typecheck": "tsc"
```
vamos a cambiarlo a solo minusculas siento que es mas facil usar este comando asi